syntax = "proto2";

message ChangeCurrentSessionDayVentillationCountMessage {
    required uint32 dayNumber = 2;
    required uint32 ventillationCount = 3;
}
